File: Difficulty: Medium/Longest Consecutive Subsequence/longest-consecutive-subsequence.java
class Solution {
    public int longestConsecutive(int[] arr) {
        // code here
        Arrays.sort(arr);
        int temp=1;
        int count=1;
        for(int i=1;i<arr.length;i++){
            if(arr[i]-arr[i-1]==0){
                continue;
            }
            if(arr[i]-arr[i-1]==1){
                temp++;
                
            }
            else{
                count=Math.max(count,temp);
                temp=1;
            }
        }
         count=Math.max(count,temp);
         return count;
    }
}

File: Difficulty: Medium/Longest Consecutive Subsequence/longest-consecutive-subsequence.py
class Solution:
    def longestConsecutive(self, arr):
        arr.sort()
        count = 1
        temp = 1
        for i in range(0,len(arr)-1):
            if arr[i+1] == arr[i] :
                continue
            elif arr[i+1] - arr[i] == 1 :
                temp = temp + 1
            else :
                count = max(temp, count)
                temp = 1
        count = max(temp,count)
        return count
       
        
         
                
             